server/database/seed/assignment_student_teacher.json

```diff
@@ -0,0 +1,46 @@
+[
+{"id":1,"student_id":1,"assignment_id":1,"is_submitted":false,"grades":1,"material_link":"http://blogspot.com/aliquam/lacus/morbi.xml?sapien=volutpat&urna=sapien&pretium=arcu&nisl=sed&ut=augue&volutpat=aliquam&sapien=erat&arcu=volutpat&sed=in&augue=congue&aliquam=etiam&erat=justo&volutpat=etiam&in=pretium&congue=iaculis&etiam=justo&justo=in&etiam=hac&pretium=habitasse&iaculis=platea&justo=dictumst&in=etiam&hac=faucibus&habitasse=cursus&platea=urna&dictumst=ut&etiam=tellus&faucibus=nulla&cursus=ut&urna=erat&ut=id&tellus=mauris&nulla=vulputate&ut=elementum&erat=nullam&id=varius&mauris=nulla&vulputate=facilisi&elementum=cras&nullam=non&varius=velit&nulla=nec&facilisi=nisi&cras=vulputate&non=nonummy&velit=maecenas&nec=tincidunt&nisi=lacus&vulputate=at&nonummy=velit&maecenas=vivamus&tincidunt=vel&lacus=nulla&at=eget&velit=eros&vivamus=elementum&vel=pellentesque&nulla=quisque&eget=porta&eros=volutpat&elementum=erat&pellentesque=quisque&quisque=erat&porta=eros&volutpat=viverra&erat=eget&quisque=congue&erat=eget&eros=semper&viverra=rutrum&eget=nulla&congue=nunc&eget=purus&semper=phasellus&rutrum=in&nulla=felis&nunc=donec"},
+{"id":2,"student_id":2,"assignment_id":2,"is_submitted":false,"grades":2,"material_link":"https://51.la/tristique/fusce.jsp?aliquam=convallis&sit=duis&amet=consequat&diam=dui&in=nec&magna=nisi&bibendum=volutpat&imperdiet=eleifend&nullam=donec&orci=ut&pede=dolor&venenatis=morbi&non=vel&sodales=lectus&sed=in&tincidunt=quam&eu=fringilla&felis=rhoncus&fusce=mauris&posuere=enim&felis=leo&sed=rhoncus&lacus=sed&morbi=vestibulum&sem=sit&mauris=amet&laoreet=cursus&ut=id&rhoncus=turpis&aliquet=integer&pulvinar=aliquet&sed=massa&nisl=id&nunc=lobortis&rhoncus=convallis&dui=tortor&vel=risus&sem=dapibus&sed=augue&sagittis=vel&nam=accumsan&congue=tellus&risus=nisi&semper=eu&porta=orci&volutpat=mauris&quam=lacinia&pede=sapien&lobortis=quis&ligula=libero&sit=nullam&amet=sit&eleifend=amet&pede=turpis&libero=elementum&quis=ligula&orci=vehicula&nullam=consequat&molestie=morbi&nibh=a&in=ipsum&lectus=integer&pellentesque=a&at=nibh&nulla=in&suspendisse=quis&potenti=justo&cras=maecenas&in=rhoncus&purus=aliquam&eu=lacus&magna=morbi&vulputate=quis"},
+{"id":3,"student_id":3,"assignment_id":3,"is_submitted":false,"grades":3,"material_link":"https://digg.com/proin.xml?curae=eu&donec=nibh&pharetra=quisque&magna=id&vestibulum=justo&aliquet=sit&ultrices=amet&erat=sapien&tortor=dignissim&sollicitudin=vestibulum&mi=vestibulum&sit=ante&amet=ipsum&lobortis=primis&sapien=in&sapien=faucibus&non=orci&mi=luctus&integer=et&ac=ultrices&neque=posuere&duis=cubilia&bibendum=curae&morbi=nulla&non=dapibus"},
```

Comment:
Just rename this file to be in camelCase and all similar files. 
Also, let this file be just `assigmentStudent` (without  `teacher`)